src/Parsers/ASTCreateQuery.h

```diff
@@ -76,7 +77,7 @@ class ASTColumns : public IAST
         f(reinterpret_cast<void **>(&primary_key));
         f(reinterpret_cast<void **>(&constraints));
         f(reinterpret_cast<void **>(&projections));
-        f(reinterpret_cast<void **>(&primary_key));
+        f(reinterpret_cast<void **>(&primary_key_from_columns));
```

Comment:
```suggestion
        f(reinterpret_cast<void **>(&primary_key));
        f(reinterpret_cast<void **>(&primary_key_from_columns));
```

Reply:
Why? There was a duplicated one already in line 76. I think it was a bug...